tests/rt_acorn.conf

```diff
@@ -1,52 +1,52 @@
-
-COMPILE | SUITES=FV3_GFS_2017_gfdlmp,FV3_CPT_v0,FV3_GSD_v0,FV3_GFS_v15_thompson,FV3_RAP,FV3_HRRR,FV3_RRFS_v1beta 32BIT=Y          | standard    |                | fv3         |
-RUN     | fv3_ccpp_gfdlmp_32bit                                                                                                   | standard    |                | fv3         |
-RUN     | fv3_ccpp_gfdlmprad_32bit_post                                                                                           | standard    |                | fv3         |
-RUN     | fv3_ccpp_cpt                                                                                                            | standard    |                | fv3         |
-RUN     | fv3_ccpp_gsd                                                                                                            | standard    |                | fv3         |
-RUN     | fv3_ccpp_thompson                                                                                                       | standard    |                | fv3         |
-RUN     | fv3_ccpp_thompson_no_aero                                                                                               | standard    |                | fv3         |
-RUN     | fv3_ccpp_rrfs_v1beta                                                                                                    | standard    |                | fv3         |
-
-################################################################################################################################################################################
-# CPLD tests                                                                                                                                                                   #
-################################################################################################################################################################################
-
-COMPILE | SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled S2S=Y                                   | standard    |                | fv3         |
-RUN     | cpld_control                                                                                                            | standard    |                | fv3         |
-RUN     | cpld_2threads                                                                                                           | standard    |                |             |
-RUN     | cpld_decomp                                                                                                             | standard    |                |             |
+##################################################################################################################################################################
```

Comment:
Do we need this to be in the repo? Do we run the test in this commit? I feel this is the same kind file as rt_stampede.conf

Reply:
I am ok to delete it, @DusanJovic-NOAA mentioned last time that acorn was somewhat special. But one can always just run a single test using `-n` or copy `rt.conf` to something else.

Reply:
We do not run any test on acorn before each commit.
Acorn (or however they call the real WCOSS2) is not the same as stampede. It will (soon?) become Tier-1 platform. Moreover it will be the NCEP's production machine.

Reply:
> I am ok to delete it, @DusanJovic-NOAA mentioned last time that acorn was somewhat special. But one can always just run a single test using `-n` or copy `rt.conf` to something else.

Ok. Delete it.

Reply:
Deleted.